unable to install subversionClient

```
  environment.systemPackages = with pkgs; [
    # ...
    subversionClient
    # ...
  ];
```

```
building path(s) ‘/nix/store/5sp708zyxh66kcw2fcwkzvq7qdqys5cb-subversion-1.9.3’, ‘/nix/store/w88qfj7sgh9kg1m60ffw6jpr7v42qryl-subversion-1.9.3-man’, ‘/nix/store/zgqa7vcnrdmax2bbvg7d5xwm65rrgyy3-subversion-1.9.3-dev’
unpacking sources
unpacking source archive /nix/store/ra3f75xy85bz9fb82xzqm796v47rwdci-subversion-1.9.3.tar.bz2
source root is subversion-1.9.3
setting SOURCE_DATE_EPOCH to timestamp 1449561201 of file subversion-1.9.3/subversion/tests/libsvn_wc/wc-test-queries.h
patching sources
applying patch /nix/store/y2pxmsc6py9zpqzikz4zr061rbphs478-apr-1.patch
patching file subversion/bindings/swig/perl/native/Makefile.PL.in
configuring
fixing libtool script ./build/ltmain.sh
configure flags: --disable-static --prefix=/nix/store/5sp708zyxh66kcw2fcwkzvq7qdqys5cb-subversion-1.9.3         --bindir=/nix/store/5sp708zyxh66kcw2fcwkzvq7qdqys5cb-subversion-1.9.3/bin --sbindir=/nix/store/5sp708zyxh66kcw2fcwkzvq7qdqys5cb-subversion-1.9.3/sbin         --includedir=/nix/store/zgqa7vcnrdmax2bbvg7d5xwm65rrgyy3-subversion-1.9.3-dev/include --oldincludedir=/nix/store/zgqa7vcnrdmax2bbvg7d5xwm65rrgyy3-subversion-1.9.3-dev/include         --mandir=/nix/store/w88qfj7sgh9kg1m60ffw6jpr7v42qryl-subversion-1.9.3-man/share/man --infodir=/nix/store/w88qfj7sgh9kg1m60ffw6jpr7v42qryl-subversion-1.9.3-man/share/info         --docdir=/nix/store/5sp708zyxh66kcw2fcwkzvq7qdqys5cb-subversion-1.9.3/share/doc/subversion         --libdir=/nix/store/5sp708zyxh66kcw2fcwkzvq7qdqys5cb-subversion-1.9.3/lib --libexecdir=/nix/store/5sp708zyxh66kcw2fcwkzvq7qdqys5cb-subversion-1.9.3/libexec         --localedir=/nix/store/5sp708zyxh66kcw2fcwkzvq7qdqys5cb-subversion-1.9.3/share/locale         --without-berkeley-db
--without-apxs
--with-swig=/nix/store/ggcj4lir4xljm2vf1p03v6nmm71wafhm-swig-3.0.7

--disable-keychain
--without-sasl
--with-serf=/nix/store/3xacrmy92xnmgpzzx2j7y4yvwlxjdmr4-serf-1.3.7
--with-zlib=/nix/store/dn9l2kd6ai731k34pzlzr5dz3in0rn31-zlib-1.2.8-dev
--with-sqlite=/nix/store/angg54wabzcrip11nhn4kr83bi4xn362-sqlite-3.12.2-dev
 
configure: Configuring Subversion 1.9.3
configure: creating config.nice
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking if gcc accepts -std=c90... yes
checking if gcc accepts -w... yes
checking if gcc accepts -Werror=unknown-warning-option... no
checking whether we are using the GNU C++ compiler... yes
checking whether g++ accepts -g... yes
checking if g++ accepts -std=c++98... yes
checking if g++ accepts -w... yes
checking if g++ accepts -Werror=unknown-warning-option... no
checking how to run the C preprocessor... gcc -E
checking for a sed that does not truncate output... /nix/store/9jj37ws95k26k8cba52a7lfz8p2f4d3l-gnused-4.2.2/bin/sed
checking build system type... x86_64-unknown-linux-gnu
checking host system type... x86_64-unknown-linux-gnu
checking target system type... x86_64-unknown-linux-gnu
checking for grep that handles long lines and -e... /nix/store/sb5x3l645wrmsvxr8k6rsdg0zwg0p8il-gnugrep-2.22/bin/grep
checking for egrep... /nix/store/sb5x3l645wrmsvxr8k6rsdg0zwg0p8il-gnugrep-2.22/bin/grep -E
checking whether ln -s works... yes
checking for a BSD-compatible install... /nix/store/7b38vz4w4rjwwww3k53h4s4ysf1ff4gx-coreutils-8.25/bin/install -c
configure: Apache Portable Runtime (APR) library configuration
checking for APR... yes
checking APR version... 1.5.2
configure: Apache Portable Runtime Utility (APRUTIL) library configuration
checking for APR-util... yes
checking APRUTIL version... 1.5.4
checking for pkg-config... no
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
configure: serf library configuration via pkg-config
configure: serf library configuration via prefix
checking serf.h usability... yes
checking serf.h presence... yes
checking for serf.h... yes
checking for serf_context_create in -lserf-1... yes
checking was serf enabled... yes
configure: looking for apr_memcache as part of apr-util
checking apr_memcache.h usability... yes
checking apr_memcache.h presence... yes
checking for apr_memcache.h... yes
checking for apr_memcache_create in -laprutil-1... yes
checking for Apache module support via DSO through APXS... no
configure: checking sqlite library
checking sqlite3.h usability... yes
checking sqlite3.h presence... yes
checking for sqlite3.h... yes
checking sqlite library version (via header)... okay
checking for sqlite3_close in -lsqlite3... yes
checking whether the compiler provides atomic builtins... yes
configure: configuring libtool now
checking how to print strings... printf
checking for a sed that does not truncate output... (cached) /nix/store/9jj37ws95k26k8cba52a7lfz8p2f4d3l-gnused-4.2.2/bin/sed
checking for fgrep... /nix/store/sb5x3l645wrmsvxr8k6rsdg0zwg0p8il-gnugrep-2.22/bin/grep -F
checking for ld used by gcc... /nix/store/ix03iknfgyrx7421fppjdczd9r4sw7pz-gcc-wrapper-5.3.0/bin/ld
checking if the linker (/nix/store/ix03iknfgyrx7421fppjdczd9r4sw7pz-gcc-wrapper-5.3.0/bin/ld) is GNU ld... yes
checking for BSD- or MS-compatible name lister (nm)... /nix/store/s0kpgl8a101lkymj599f5a04s4wflf5s-binutils-2.26-dev/bin/nm -B
checking the name lister (/nix/store/s0kpgl8a101lkymj599f5a04s4wflf5s-binutils-2.26-dev/bin/nm -B) interface... BSD nm
checking the maximum length of command line arguments... 1572864
checking how to convert x86_64-unknown-linux-gnu file names to x86_64-unknown-linux-gnu format... func_convert_file_noop
checking how to convert x86_64-unknown-linux-gnu file names to toolchain format... func_convert_file_noop
checking for /nix/store/ix03iknfgyrx7421fppjdczd9r4sw7pz-gcc-wrapper-5.3.0/bin/ld option to reload object files... -r
checking for objdump... objdump
checking how to recognize dependent libraries... pass_all
checking for dlltool... no
checking how to associate runtime and link libraries... printf %s\n
checking for ar... ar
checking for archiver @FILE support... @
checking for strip... strip
checking for ranlib... ranlib
checking for gawk... gawk
checking command to parse /nix/store/s0kpgl8a101lkymj599f5a04s4wflf5s-binutils-2.26-dev/bin/nm -B output from gcc object... ok
checking for sysroot... no
checking for a working dd... /nix/store/7b38vz4w4rjwwww3k53h4s4ysf1ff4gx-coreutils-8.25/bin/dd
checking how to truncate binary pipes... /nix/store/7b38vz4w4rjwwww3k53h4s4ysf1ff4gx-coreutils-8.25/bin/dd bs=4096 count=1
./configure: line 9842: /usr/bin/file: No such file or directory
checking for mt... no
checking if : is a manifest tool... no
checking for dlfcn.h... yes
checking for objdir... .libs
checking if gcc supports -fno-rtti -fno-exceptions... no
checking for gcc option to produce PIC... -fPIC -DPIC
checking if gcc PIC flag -fPIC -DPIC works... yes
checking if gcc static flag -static works... no
checking if gcc supports -c -o file.o... yes
checking if gcc supports -c -o file.o... (cached) yes
checking whether the gcc linker (/nix/store/ix03iknfgyrx7421fppjdczd9r4sw7pz-gcc-wrapper-5.3.0/bin/ld) supports shared libraries... yes
checking whether -lc should be explicitly linked in... no
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... no
checking how to run the C++ preprocessor... g++ -E
checking for ld used by g++... /nix/store/ix03iknfgyrx7421fppjdczd9r4sw7pz-gcc-wrapper-5.3.0/bin/ld
checking if the linker (/nix/store/ix03iknfgyrx7421fppjdczd9r4sw7pz-gcc-wrapper-5.3.0/bin/ld) is GNU ld... yes
checking whether the g++ linker (/nix/store/ix03iknfgyrx7421fppjdczd9r4sw7pz-gcc-wrapper-5.3.0/bin/ld) supports shared libraries... yes
checking for g++ option to produce PIC... -fPIC -DPIC
checking if g++ PIC flag -fPIC -DPIC works... yes
checking if g++ static flag -static works... no
checking if g++ supports -c -o file.o... yes
checking if g++ supports -c -o file.o... (cached) yes
checking whether the g++ linker (/nix/store/ix03iknfgyrx7421fppjdczd9r4sw7pz-gcc-wrapper-5.3.0/bin/ld) supports shared libraries... yes
checking dynamic linker characteristics... (cached) GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
configure: building shared libraries only
checking whether libtool needs -no-undefined... no
checking for trang... none
checking for doxygen... none
checking for Expat... yes
checking for socket in -lsocket... no
checking whether to look for SASL... no
checking for Mach-O dynamic module iteration functions... no
checking for Mac OS property list utilities... no
checking for Mac OS KeyChain Services... checking whether APR has support for DSOs... yes
checking whether to support GPG-Agent... yes
checking whether to look for GNOME Keyring... yes
checking whether use Googlemock... no
checking for msgfmt... none
checking for msgmerge... none
checking for xgettext... none
checking magic.h usability... no
checking magic.h presence... no
checking for magic.h... no
checking whether to look for KWallet... no
checking for ANSI C header files... (cached) yes
checking for an ANSI C-conforming const... yes
checking for size_t... yes
checking for working memcmp... yes
checking for vprintf... yes
checking for _doprnt... no
checking for symlink... yes
checking for readlink... yes
checking sys/utsname.h usability... yes
checking sys/utsname.h presence... yes
checking for sys/utsname.h... yes
checking for uname... yes
checking termios.h usability... yes
checking termios.h presence... yes
checking for termios.h... yes
checking for tcgetattr... yes
checking for tcsetattr... yes
checking for -Wl,--no-undefined... yes
configure: C compiler flags: -g -O2  -g -O2 -pthread
configure:     user-defined: 
configure:  maintainer-mode: 
configure: C++ compiler flags: -g -O2
configure:       user-defined: 
configure:    maintainer-mode: 
configure: zlib library configuration
checking zlib.h usability... yes
checking zlib.h presence... yes
checking for zlib.h... yes
checking for inflate in -lz... yes
checking for python... /nix/store/adnbhcy51b4x19fmfci6h0qfy5sarwpr-python-2.7.11/bin/python
checking for JDK... no
checking for perl... /nix/store/72bpwv0lf1xyrysmz6bh4sldmynzdxq2-perl-5.22.1/bin/perl
checking for ruby... no
checking for ruby1.8... no
checking for ruby18... no
checking for ruby1.9... no
checking for ruby1... no
checking for ruby1.9.3... no
checking for ruby193... no
checking for ruby2.0... no
checking for ruby2.1... no
checking swig version... 3.0.7
configure: WARNING: Detected SWIG version 3.0.7
configure: WARNING: Subversion requires SWIG >= 1.3.24 and < 3.0.0 
configure: Configuring python swig binding
checking for Python includes... -I/nix/store/adnbhcy51b4x19fmfci6h0qfy5sarwpr-python-2.7.11/include/python2.7
checking for compiling Python extensions... gcc -pthread -fPIC
checking for linking Python extensions... gcc -pthread -shared -lgcc_s
checking for linking Python libraries... -lgcc_s
checking for apr_int64_t Python/C API format string... 
configure: error: failed to recognize APR_INT64_T_FMT on this platform
builder for ‘/nix/store/xn56ad5499wz6irmpmcpffi4g2hhl016-subversion-1.9.3.drv’ failed with exit code 1
cannot build derivation ‘/nix/store/3cx5y9pnp4jiac20yp4dm91pj9447cn1-system-path.drv’: 1 dependencies couldn't be built
cannot build derivation ‘/nix/store/4ja91xnb5ysj73dvhndnvvdaj4fc7j1v-nixos-system-xps-16.09pre83100.25e3c09.drv’: 1 dependencies couldn't be built
error: build of ‘/nix/store/4ja91xnb5ysj73dvhndnvvdaj4fc7j1v-nixos-system-xps-16.09pre83100.25e3c09.drv’ failed
```

The platform is x86_64. I'm not sure why nix thought I needed to compile subversion client instead of downloading it from Hydra.

## Technical details

* System: 16.09pre83100.25e3c09 (Flounder)
* Nix version: nix-env (Nix) 1.11.2

--- a/pkgs/applications/version-management/subversion/default.nix
+++ b/pkgs/applications/version-management/subversion/default.nix
@@ -37,6 +37,10 @@ let
 
     patches = [ ./apr-1.patch ];
 
+    # SVN build seems broken on gcc5:
+    # https://gcc.gnu.org/gcc-5/porting_to.html
+    CPPFLAGS = "-P";
+
     configureFlags = ''
       ${if bdbSupport then "--with-berkeley-db" else "--without-berkeley-db"}
       ${if httpServer then "--with-apxs=${apacheHttpd}/bin/apxs" else "--without-apxs"}

